components/modal/modal.scss

```diff
@@ -133,6 +131,7 @@
 .ui-modal__content {
   -webkit-overflow-scrolling: touch;
   background-color: var(--tx-modal-content-background-color);
+  max-height: calc(54vh);
```

Comment:
Why do you need `calc` here? Just `54vh` is enough. Another question is why it's exactly 54?

Reply:
It's not really critical, but It's would be nice to change the layout for `.ui-modal__container` and make it `display: flex`. By that, we can specify `max-height` for that main container only and children will be settled automatically. Right now I don't see any option than set up some fixed value (same as you did, but without `calc` please 😄 )

Reply:
@yurist38 
mmm, just doing some explorations because max-height is not working as expected in IE
https://github.com/philipwalton/flexbugs/issues/216

Reply:
Okay, I see. I'm fine with this solution for now once you have removed `calc()`... 😄